src/Engines/Processors/shaderProcessor.ts

```diff
@@ -9,6 +9,8 @@ import { ShaderDefineExpression } from './Expressions/shaderDefineExpression';
 import { ShaderDefineArithmeticOperator } from './Expressions/Operators/shaderDefineArithmeticOperator';
 import { ProcessingOptions } from './shaderProcessingOptions';
 import { _DevTools } from '../../Misc/devTools';
+import { ShaderLanguage } from "../../Materials/shaderLanguage";
+import { CodeStringParsingTools } from "../../Misc/codeStringParsingTools";

```

Comment:
This as lots of WebGPUSpecific in growing the size of ThinEngine bundles, so either you split the file, or in the file, do not create a class and simply export the functions separately so tree shakers will only grab what they truly use :-)

Reply:
I have done something for that, let me know if it's ok for you.